schemas/com.snowplowanalytics.monitoring/sink_write_failed/jsonschema/1-0-0

```diff
@@ -0,0 +1,37 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a write to a sink failing",
+	"self": {
+		"vendor": "com.snowplowanalytics.monitoring",
+		"name": "sink_write_failed",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+
+	"properties": {
+
+		"sink": {
+			"type": "string"
+		},
+
+		"failureCount": {
+			"type": "integer"
+		},
+
+		"initialFailureTime": {
+			"type": "number"
+		},
+
+		"lastRetryPeriod": {
+			"type": "number"
```

Comment:
Good point!

I think failureCount can be an integer since we are unlikely to have 2147483647 failures.
But initialFailureTime is in ms since the Unix epoch, i.e. about 1 435 234 971 319, which is bigger than the Redshift int4 maximum.

So it sounds like I need to change the Redshift type to bigint like it is for performance timing.

The types can all be integer rather than number and I can add the -9223372036854775808 to 9223372036854775807 constraint to the schema.